internal/pkg/dashboard/components/components.go

```diff
@@ -75,14 +75,26 @@ func formatStatus(status any) string {
 
 	switch strings.ToLower(statusStr) {
 	case "running", "healthy", "true":
-		return fmt.Sprintf("[green]%s[-]", statusStr)
+		return formatText(statusStr, true)
 	case "stopped", "unhealthy", "false":
-		return fmt.Sprintf("[red]%s[-]", statusStr)
+		return formatText(statusStr, false)
 	default:
 		return statusStr
 	}
 }
 
+func formatText(text string, status bool) string {
+	if text == "" {
+		return ""
+	}
+
+	if status {
+		return fmt.Sprintf("[green]\u221A %s[-]", text)
```

Comment:
we can use the actual character here, it's Go, source is UTF-8!

this will make it a bit more readable